api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java

```diff
@@ -369,7 +376,22 @@ public ParticipantDemographics findParticipantDemographics() {
         .collect(Collectors.toList());
   }
 
+  private void validateSearchTerm(String term) {
+    if (StringUtils.isEmpty(term)) {
```

Comment:
Question: Should we add check to see if term contains characters say like %?

Reply:
The UI is currently doing this validation. We can also add it here if we think this is needed. If so we can create a new ticket for the work with a detailed description on what to validate.